Makefile

```diff
@@ -23,6 +23,7 @@ publish:
 		echo "now must:"; \
 		echo "- seed websites: /ipfs/$$hash"; \
 		echo "- add to ipfs.io: $$ipfshash"; \
+	dnslink-deploy --domain=ipfs.io --record=@ --path=$$ipfshash
```

Comment:
maybe this could be another target:

```
publish-to-domain: publish auth.token
    TOKEN=$(shell cat auth.token) dnslink-deploy --domain=ipfs.io --record=@ --path=$$ipfshash
```
(not set at all on `auth.token` filename)